pkgs/data/fonts/google-fonts/default.nix

```diff
@@ -30,7 +30,11 @@ stdenv.mkDerivation {
     rm -rv ofl/comfortaa/*.ttf \
       ofl/mavenpro/*.ttf \
       ofl/muli/*.ttf \
-      ofl/oswald/*.ttf
+      ofl/oswald/*.ttf \
+
+    # This abomination of a font causes crashes with `libfontconfig',
+    # It has an absurd number of symbols
+    rm -r ofl/adobeblank/
```

Comment:
Maybe we could add `outputs = [ "out" "adobeBlank" ];` after the version attribute and use `moveToOutput share/fonts/truetype/something.ttf "$adobeBlank"` where `something.ttf` are the file names of Adobe Blank font files in `installPhase`.

That way it will not be installed by default but interested people will still be able to install it using `google-fonts.adobeBlank`.

Reply:
That sounds like a good plan.

Reply:
I tried using `moveToOutput` during `installPhase`, but it wasn't working as expected.
I went ahead and did it the ol' fashioned way.

`rm` was necessary to prevent the font from being installed to `$out`.

I have tested `google-fonts` and `google-fonts.adobeBlank`, and they work as expected.